### PR TITLE
feat: support js-cookie attributes

### PR DESCRIPTION
**Pull request by Gomah**
Introduces cookie Attributes support from the `js-cookie` package.

Unfortunately, it also introduces a breaking change for the `tokenExpiry` option.

### CODE REVIEW

**Comment by dohomi** [comment]
@Gomah could you prepare this PR that you proof on `typeof tokenExpiry === Number` to have a fallback to the old behaviour? Then include `js-cookie` only on demand if needed? Then both versions will be working without breaking changes ( currently its a RC and I would like to avoid breaking changes to the API now Vue is SSR ready )

**Comment by Gomah** [comment]
@dohomi Just added the fallback for the `tokenExpiry` option, maybe we should add a warning too? 

> Then include `js-cookie` only on demand if needed?

I can create a separate PR for this one if needed but `js-cookie` was already imported in plugin.js.

**Comment by kieusonlam** [comment]
Hi @Gomah 

It would be great if we can add added the fallback and deprecation warnings. Can you still work on this?

**Comment by Gomah** [comment]
@kieusonlam Just got time look at it, should be good now :v:

File: lib/templates/plugin.js [hunk 1]
```diff
@@ -102,11 +104,11 @@ export default (ctx, inject) => {
   }
 
   inject('apolloHelpers', {
-    onLogin: async (token, apolloClient = apolloProvider.defaultClient, tokenExpires = AUTH_TOKEN_EXPIRES) => {
+    onLogin: async (token, apolloClient = apolloProvider.defaultClient, cookieAttributes = COOKIE_ATTRIBUTES) => {
```
[the remark this answers is not in the document]
Comment (Gomah) [reply lib/templates/plugin.js]: Good catch, fixed.

**Comment by kieusonlam** [comment]
Thanks

**Pull request merged**